soc/arm/nuvoton_npcx/npcx7/Kconfig.defconfig.series

```diff
@@ -26,6 +26,10 @@ config GPIO_NPCX
 	default y
 	depends on GPIO
 
+config PINCTRL_NPCX
+	default y
+	depends on PINCTRL
```

Comment:
this is not required when defaulting based on DT status, please check previous comment.

Reply:
Done.